prisma/seed.ts

```diff
@@ -0,0 +1,204 @@
+//TODO - add npx prisma db seed command to docs
+
+import { PrismaClient } from "@prisma/client";
+
+const prisma = new PrismaClient();
+
+async function seedingDatabase() {
```

Comment:
just a little thing - verb in present simple
```suggestion
async function seedDatabase() {
```

Reply:
Sure, will fix that